libqtile/layout/tree.py

```diff
@@ -752,6 +758,7 @@ def layout(self, windows: Sequence[base.Window], screen_rect: ScreenRect) -> Non
         Layout.layout(self, windows, body)
 
     def _resize_panel(self, screen_rect):
+        print("Resize Panel")
```

Comment:
Remove this, otherwise looks good

Reply:
I need to add my print statement checker as a pre-commit hook. I have it on qtile-extras too prevent annoying things like this!